spec/factories/state_file_md_intakes.rb

```diff
@@ -118,6 +124,16 @@
       spouse_last_name { "Lando" }
     end
 
+    factory :state_file_md_refund_intake do
```

Comment:
Why do you need this factory case? And if you do need i would advise adding it as a trait instead so it would build or overwrite the base intake something more like:
```suggestion
    trait :with_refund do
    after(:build) do |intake, evaluator|
        intake.direct_file_data.fed_wages = 2_000
        intake.direct_file_data.fed_taxable_income = 2_000
        intake.direct_file_data.fed_taxable_ssb = 0
        intake.direct_file_data.fed_unemployment = 0
      end
    end
```